src/components/icon/_appstore.less

```diff
@@ -66,7 +66,7 @@
       position: absolute;
       content: "Get it on";
       top: 3px;
-      left: 100px;
+      left: 87px;
```

Comment:
Don't change it for all buttons just change it for buttons that have the class `.block-btn`

Reply:
yes but this change will only reflect on block btn because for non block button style is left: 50px  ; so indirectly it is only for block btns ... I guess